omnibus/config/projects/agent.rb

```diff
@@ -251,7 +251,23 @@
 exclude '\.git*'
 exclude 'bundler\/git'
 
-if linux?
+if windows?
+  #
+  # For Windows build, files need to be stripped must be specified here.
+  #
+  windows_files_to_strip [
+    "#{Omnibus::Config.source_dir()}\\cf-root\\bin\\agent\\security-agent.exe",
+    "#{Omnibus::Config.source_dir()}\\cf-root\\bin\\agent\\process-agent.exe",
+    "#{Omnibus::Config.source_dir()}\\cf-root\\bin\\agent\\trace-agent.exe",
+    "#{Omnibus::Config.source_dir()}\\cf-root\\bin\\agent.exe",
+    "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\security-agent.exe",
+    "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\process-agent.exe",
+    "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\trace-agent.exe",
+    "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\agent.exe"
```

Comment:
Do you know why each binary is there twice?

Reply:
It's due to the way we build MSI and ZIP. They are coming from difference locations.